src/executables/util_cs.c

```diff
@@ -5092,9 +5097,23 @@ flashback (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
-  printf ("Enter transaction id : ");
-  scanf ("%d", &trid);
+  {
+    POLL_FD input_fd = { STDIN_FILENO, POLLIN | POLLPRI };
+    int timeout = prm_get_integer_value (PRM_ID_FLASHBACK_TIMEOUT);
```

Comment:
1. PRM_ID_FLASHBACK_TIMEOUT is just for flashback utility, not for other clients. So I think it should be a utility option, not a system parameter.
2. One concern is that this is a client-side parameter (or option), so one client can monopolize the flashback with a very long timeout. But this utility is only executed by DBA, so the problem is alleviated.